envbuilder.go

```diff
@@ -77,152 +76,9 @@ var (
 	MagicFile = filepath.Join(MagicDir, "built")
 )
 
-type Options struct {
```

Comment:
I'd like to preserve the structure for Options as it ensures type safety. You don't need to use `GetString()`, `GetBool()` methods as Go can handle this natively.

Reply:
Hum.. how I can preserve the `Options` structure and add a description for each item? I talked to @mafredri about having the descriptions on the CLI only but making this we guess developers are not going to keep it updated since it would be in a different place where the option is defined.

Reply:
Since we want to use `coder/serpent` for this I think the `Options`  will need to change anyway.